lib/RuntimePlugin.js

```diff
@@ -413,6 +414,15 @@ class RuntimePlugin {
 					);
 					return true;
 				});
+			compilation.hooks.runtimeRequirementInTree
+				.for(RuntimeGlobals.baseURI)
+				.tap("RuntimePlugin", chunk => {
+					const { chunkLoading } = chunk.getEntryOptions();
+					if (chunkLoading === false) {
```

Comment:
This needs a similar logic as https://github.com/webpack/webpack/blob/770dea1fb46c7c5e04016dbab1f0854037b00d81/lib/esm/ModuleChunkLoadingPlugin.js#L24-L32

to handle the global `output.chunkLoading: false` too.